templates/azure-pipelines.yml

```diff
@@ -47,94 +47,94 @@ stages:
       vmImage: 'ubuntu-16.04'
     strategy:
       matrix:
-        GCC 8:
+        GCC_8:
           B2_TOOLSET: gcc
           B2_CXXSTD: 14,17,2a
           CXX: g++-8
           PACKAGES: g++-8
-        GCC 7:
+        GCC_7:
           B2_TOOLSET: gcc
           B2_CXXSTD: 11,14,17
           CXX: g++-7
           PACKAGES: g++-7
-        GCC 6:
+        GCC_6:
           B2_TOOLSET: gcc
           B2_CXXSTD: 11,14
           CXX: g++-6
           PACKAGES: g++-6
-        GCC 5:
+        GCC_5:
           B2_TOOLSET: gcc
           B2_CXXSTD: 11
           CXX: g++-5
           PACKAGES: g++-5
-        GCC 4.9:
+        GCC_4.9:
           B2_TOOLSET: gcc
           B2_CXXSTD: 03,11
           CXX: g++-4.9
           PACKAGES: g++-4.9
-        GCC 4.8:
+        GCC_4.8:
```

Comment:
Missing dot replacement

Reply:
Thanks for catching this and the other ones. Fixed now.